### PR TITLE
feat: video listing screen added

### PR DESCRIPTION
**Pull request by JowelTisso**
This PR is for video listing page:
Main file : 
* VideoListing.jsx and  VideoListing.css
* VideoReducer.js and VideoProvider.js
* VideoCard.jsx and VideoCard.css

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -5,6 +5,7 @@ import App from "./App";
 import { makeServer } from "./server";
 import { BrowserRouter } from "react-router-dom";
 import { AuthProvider } from "./src/context/provider/AuthProvider";
+import { VideoProvider } from "./src/context/provider/VideoProvider";
```
Comment (jeetsdev) [inline src/index.js:8]: You can use barrel export here for all the contexts.

**Review by jeetsdev — APPROVED** [review]
- I like how you have used constants rather strings.
-  Good folder structure 
- Self-declarative names

Just a suggestion to use barrel export for clean code.

File: src/src/components/video/VideoCard.jsx [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import "./VideoCard.css";
+
+const VideoCard = ({ img, title, artist, views }) => {
+  return (
+    <div className="video-card pointer">
+      <img src={img} alt="thumbnail" className="video-thumb mg-top-2x" />
```
Comment (praveenreddy1798) [inline src/src/components/video/VideoCard.jsx]: can add lazy loading to image

File: src/src/pages/videos/videoListing/VideoListing.jsx [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+        </button>
+      </div>
+      <main className="videos-container">
+        {videos.map(({ _id, videoId, title, thumbnail, artist }) => (
+          <Link to={`/videoplayer/${videoId}`} key={_id} className="no-deco">
+            <VideoCard
+              img={thumbnail}
+              title={title}
+              artist={artist}
+              views={"40M"}
```
Comment (praveenreddy1798) [inline src/src/pages/videos/videoListing/VideoListing.jsx]: Why is the views hardcoded ?

**Comment by praveenreddy1798** [comment]
Overall I like how you have structured your code. Use barrel imports as suggested by jeet and also added few comments that can make ur code better.

File: src/index.js (hunk 1 above)
Comment (JowelTisso) [reply src/index.js:8]: Ok will try that, Thanks.

File: src/src/components/video/VideoCard.jsx (hunk 2 above)
Comment (JowelTisso) [reply src/src/components/video/VideoCard.jsx]: Ok will try that

File: src/src/pages/videos/videoListing/VideoListing.jsx (hunk 3 above)
Comment (JowelTisso) [reply src/src/pages/videos/videoListing/VideoListing.jsx]: Cause I haven't added the value in database, will do it.

**Pull request merged**